tests/unit_tests/service_nodes.cpp

```diff
@@ -356,28 +359,81 @@ TEST(service_nodes, min_portions)
   {
     const auto large = 4 * (STAKING_PORTIONS / 5);
     const auto rest = STAKING_PORTIONS - large;
-    bool result = service_nodes::check_service_node_portions({large, rest});
+    bool result = service_nodes::check_service_node_portions(hf_version, {large, rest});
     ASSERT_TRUE(result);
   }
 
   // Three contributers
   {
     const auto half = STAKING_PORTIONS / 2 - 1;
     const auto rest = STAKING_PORTIONS - 2 * half;
-    bool result = service_nodes::check_service_node_portions({half, half, rest});
+    bool result = service_nodes::check_service_node_portions(hf_version, {half, half, rest});
     ASSERT_TRUE(result);
   }
 
   // Four contributers
   {
     const auto third = STAKING_PORTIONS / 3 - 1;
     const auto rest = STAKING_PORTIONS - 3 * third;
-    bool result = service_nodes::check_service_node_portions({third, third, third, rest});
+    bool result = service_nodes::check_service_node_portions(hf_version, {third, third, third, rest});
+    ASSERT_TRUE(result);
+  }
+
+  // ===== After hard fork v11 =====
+  hf_version = cryptonote::network_version_11_swarms;
```

Comment:
It's also worth testing all the previous cases before still work after hardfork 11.